assembl/static2/js/app/graphql/IdeaWithPostsQuery.graphql

```diff
@@ -11,6 +11,13 @@ query IdeaWithPosts($id: ID!) {
               id
               parentId
               creationDate
+              publicationState
```

Comment:
So all logic for posts are pushed to IdeaWithPostQuery?

Reply:
It contains what we need to create the tree structure (id, parentId),  sort (creationDate, future feature), sum of sentiments (sentimentCounts, publicationState).